src/pages/CategoriesPage/AddCategory/AddNewCategoryComponent.tsx

```diff
@@ -7,73 +7,86 @@ import {
-            <div style={styles.fieldsWrapper}>
-              <TextField
-                id="name"
-                label="Category Name"
-                variant="outlined"
-                onChange={formProps.handleChange}
-                value={formProps.values.name}
-                error={formProps.touched.name && Boolean(formProps.errors.name)}
-                helperText={formProps.touched.name && formProps.errors.name}
-              />
-            </div>
-            <div style={styles.fieldsWrapper}>
-              <TextField
-                id="description"
-                label="Description"
-                variant="outlined"
-                onChange={formProps.handleChange}
-                value={formProps.values.description}
-                error={
-                  formProps.touched.description &&
-                  Boolean(formProps.errors.description)
-                }
-                helperText={
-                  formProps.touched.description && formProps.errors.description
-                }
-              />
-            </div>
-            <Grid container>
-              <Grid item xs={6}>
-                <div style={styles.buttonsWrapper}>
-                  <Button
-                    sx={styles.closeButton}
-                    onClick={props.onToggleCreateStatusClick}
-                  >
-                    Close
-                  </Button>
-                </div>
+        <Formik
+          initialValues={{
+            name: '',
+            description: '',
+          }}
+          validationSchema={validationSchema}
+          onSubmit={(values: CreateUpdateCategoryModel) => {
+            props.onSubmit(values);
+          }}
+        >
+          {(formProps) => (
+            <form onSubmit={formProps.handleSubmit}>
+              <div style={styles.fieldsWrapper}>
+                <TextField
+                  id="name"
+                  label="Category Name"
+                  variant="outlined"
+                  onChange={formProps.handleChange}
+                  value={formProps.values.name}
+                  error={
+                    formProps.touched.name && Boolean(formProps.errors.name)
+                  }
+                  helperText={formProps.touched.name && formProps.errors.name}
+                />
+              </div>
+              <div style={styles.fieldsWrapper}>
+                <TextField
+                  id="description"
+                  label="Description"
+                  variant="outlined"
+                  onChange={formProps.handleChange}
+                  value={formProps.values.description}
+                  error={
+                    formProps.touched.description &&
+                    Boolean(formProps.errors.description)
+                  }
+                  helperText={
+                    formProps.touched.description &&
+                    formProps.errors.description
+                  }
+                />
+              </div>
```

Comment:
Do not use DIV. Use Box instead.

Reply:
All divs in the modal windows have been replaced by the Box components